src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorColor.java

```diff
@@ -51,34 +67,87 @@ public void initializeConverter() {
 
         clusterColorControl = (ZclColorControlCluster) endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID);
         if (clusterColorControl == null) {
-            logger.error("Error opening device control controls {}", endpoint.getIeeeAddress());
+            logger.error("Error opening device color controls {}", endpoint.getIeeeAddress());
             return;
         }
 
         clusterLevelControl = (ZclLevelControlCluster) endpoint.getInputCluster(ZclLevelControlCluster.CLUSTER_ID);
         if (clusterLevelControl == null) {
-            logger.error("Error opening device level controls {}", endpoint.getIeeeAddress());
-            return;
+            logger.warn("Device does not support level control {}", endpoint.getIeeeAddress());
         }
 
-        clusterColorControl.bind();
-        clusterLevelControl.bind();
+        clusterOnOff = (ZclOnOffCluster) endpoint.getInputCluster(ZclOnOffCluster.CLUSTER_ID);
+        if (clusterOnOff == null) {
+            logger.warn("Device does not support on/off control {}", endpoint.getIeeeAddress());
```

Comment:
Same as above, in terms of war vs debug

Reply:
Is On/Off NEEDED? In the past it wasn't as we used the MoveWithOnOff method in the level control. This still looks to be the case, so On/Off will still work, but using the level control (right?). If so, I would say this is debug as no need to confuse the user when ON/OFF will work just fine.

Reply:
Agreed, as the implemetation will default to level control if supported and if it is not then it will already warn about it (as per above). So moving this to debug